calico-policies/public-network-isolation/us-south/allow-public-services-pods.yaml

```diff
@@ -37,14 +37,15 @@ spec:
   - action: Allow
     destination:
       nets:
-      # IBM Cloud Monitoring with Sysdig: US South nets
+      # IBM Cloud Monitoring with Sysdig: us-south
+      # Taken from: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints
+      - 169.60.151.174/32
       - 169.46.0.70/32
-      - 169.47.102.30/32
-      - 169.47.227.243/32
       - 169.48.214.70/32
-      - 169.60.151.174/32
-      - 169.62.255.114/32
-      - 169.62.255.116/32
+      # IBM Cloud Monitoring with Sysdig: br-sao subnets
+      - 163.107.66.98/32
```

Comment:
Why do us-south clusters need to access br-sao sysdig IPs? I don't see this mentioned in the docs specified here: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints

Reply:
@bradbehle , when deploying to single zone br-sao, we are using us-south policies as mentioned in the note. That is the reason

Reply:
Oh, I forgot about those single zone regions.  Thanks